src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.MoveFile.cs

```diff
@@ -0,0 +1,238 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Threading;
+using Xunit;
+
+public partial class FileSystemWatcher_4000_Tests
+{
+    #region WindowsTests
+
+    [Theory]
+    [InlineData(WatcherChangeTypes.Changed, "changed", false)]
```

Comment:
Yes, they are a bit slower as there are a number of false events. On my Windows machine, it takes 8 seconds to run all of the FSW tests. 

Do you think that is fine, or should we trim it down a bit? I think the SameDirectory, NestedDirectory, and  NotifyFilter tests need to stay but I'd be okay removing the Different_Directory tests if necessary. We could outerloop them.